### PR TITLE
Add tag website build text to README.md

### PR DESCRIPTION
**Pull request by postscript-dev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -398,6 +399,39 @@ To build the documentation, you must install the following products:
...
+For example,
+
+```bash
+$ export EXIV2HOME=<exiv2dir>
+```
+
+Then, change to the `doc/templates` directory and run `make`.
+
+```bash
+$ cd <exiv2dir>/doc/templates
```
Comment (piponazo) [inline README.md:428]: I tried to follow the steps and I discovered something that could be improved. The Makefile under the templates folder uses the EXIV2HOME variable like this:

```
TAGLIST = ${EXIV2HOME}/build/bin/taglist
```
So it is assuming that the `build` directory has that specific name. I think it would be better to use a variable EXIV2_BINDIR as we do when running the python tests, so that the script can easily find the `taglist` executable. At the moment, even if you follow the instructions step by step, the script does not have a way to know what's your build directory.

Another thing that I noticed is that the Makefile is assuming the existence of some prerequisites: bash, make, xsltproc, python3. It would be nice to mention in this README those depedencies.

File: README.md [hunk 2]
```diff
@@ -398,6 +399,39 @@ To build the documentation, you must install the following products:
...
+$ make
+```
+
+After processing, the generated webpages are stored in the `<exiv2dir>/doc/templates` directory. 
+When the Exiv2 websites are updated, the generated tag webpages are reformatted before use.
```
Comment (piponazo) [inline README.md:433]: After running that Makefile the directory gets filled with all the generated files ... and it is tedious to delete them to get your local repository to a clean state. I think it would be nice to generate these files under a new sub-directory so that one can delete that directory afterwards easily. 

Maybe we can do that in another PR

**Review by piponazo — COMMENTED** [review]
Thanks for the new piece of doc :clap: 

I made some comments, but they might be out of scope for this PR. Let me know if you are willing to address some of them, or otherwise I am fine with merging this as it is.

File: README.md (hunk 1 above)
Comment (postscript-dev) [reply README.md:428]: @piponazo:
Thanks, I have made the updates.

File: README.md (hunk 2 above)
Comment (postscript-dev) [reply README.md:433]: I agree, it would be easier to have the files separated out but I think that I will leave this for a different PR.

**Review by piponazo — APPROVED** [review]
Thanks for taking care of the suggestions 👍
